cdap-hbase-compat-0.96/src/main/java/co/cask/cdap/data2/util/hbase/HBase96TableUtil.java

```diff
@@ -92,12 +91,10 @@ public void createNamespaceIfNotExists(HBaseDDLExecutor ddlExecutor, String name
   }
 
   @Override
-  public void deleteNamespaceIfExists(HBaseAdmin admin, String namespace) throws IOException {
-    Preconditions.checkArgument(admin != null, "HBaseAdmin should not be null");
+  public void deleteNamespaceIfExists(HBaseDDLExecutor ddlExecutor, String namespace) throws IOException {
```

Comment:
similar comment, can we remove this in favor of just using the ddlExecutor to delete?